src/components/Todolist.tsx

```diff
@@ -0,0 +1,80 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { Todo } from '../types/Todo';
+import { Form } from './Form';
+import { TodoModal } from './TodoModal';
+
+type Props = {
+  title: string;
+  todos: Todo[];
+  setTitle: (title: string) => void;
+};
+
+export const TodoList: React.FC<Props> = ({
+  title,
+  todos,
+  setTitle,
+}) => {
+  const [editing, setEditing] = useState(false);
+
+  return (
+    <section className="todoapp__main">
+      {todos.map(todo => (
+        <div
+          key={todo.id}
+          className={classNames(
+            'todo',
+            { completed: todo.completed },
+          )}
+        >
+          <label
+            className="todo__status-label"
+          >
+            <input
+              type="checkbox"
+              className="todo__status"
+              id="todo__selected"
+            />
```

Comment:
The input element has a static id attribute. It is not recommended to have the same id for multiple elements. Consider generating a unique id for each todo using todo.id.